stdlib/urllib/request.pyi

```diff
@@ -155,6 +155,8 @@ class HTTPPasswordMgrWithPriorAuth(HTTPPasswordMgrWithDefaultRealm):
 
 class AbstractBasicAuthHandler:
     rx: ClassVar[Pattern[str]]  # undocumented
+    passwd: Union[HTTPPasswordMgr, None]
+    add_password: Callable[[], Any]
```

Comment:
This has the same arguments and return value as the `add_password` method in `HTTPPasswordMgr`. Please add them here.

Reply:
However, this would prevent the user to do something like:

```python
import urllib.request
authinfo = urllib.request.HTTPBasicAuthHandler()
addpassword = authinfo.add_password
```

right?

Reply:
And then make a call.

Reply:
That code will work either way. But note that `addpassword()` without any arguments should be an error, because it needs arguments.